### PR TITLE
Code cleanup

### PR DESCRIPTION
**Pull request by userMacieG**
Reformat code, replace `RegisterServerEvent` with `RegisterNetEvent`, remove `AddEventHandler` (FiveM is adding that with `RegisterNetEvent`), moved playertables to config, replace spaces to tabs, fix formatting

### CODE REVIEW

**Comment by BerkieBb** [comment]
There is a double `lua54 'yes'` in the fxmanifest.lua

**Comment by userMacieG** [comment]
> There is a double `lua54 'yes'` in the fxmanifest.lua

Fixed, also do you want this type of pr?

**Comment by GhzGarage** [comment]
Yes, PR’s like this are much needed so thank you. I’m going to dive into this in the morning hopefully and get it merged but from what I can tell it all looks fine

**Review by FreakOutModz — APPROVED** [review]
This is a clean PR, but might broke some event maybe. Not recommend for merge before testing in multiple QB servers. Just replacing every `PlayerPedId()` will help a lot. Nice PR!

**Comment by phumberdroz** [comment]
Would you mind resolving the Merge conflicts?  :)

File: server/loops.lua [hunk 1]
```diff
@@ -1,13 +1,15 @@
...
-    for i=1, #Players, 1 do
-        local Player = QBCore.Functions.GetPlayer(Players[i])
-
-        if Player.PlayerData.job ~= nil and Player.PlayerData.job.payment > 0 then
-            Player.Functions.AddMoney('bank', Player.PlayerData.job.payment)
-            TriggerClientEvent('QBCore:Notify', Players[i], "You received your paycheck of $"..Player.PlayerData.job.payment)
-        end
-    end
-    SetTimeout(QBCore.Config.Money.PayCheckTimeOut * (60 * 1000), PaycheckLoop)
+function PaycheckLoop()
```
Comment (FreakOutModz) [inline server/loops.lua:1]: Please keep function code methot used by QBCore `PaycheckLoop = function()`

File: server/functions.lua [hunk 2]
```diff
@@ -223,31 +260,39 @@ end
...
-    end
+	local result = exports.ghmattimysql:executeSync('SELECT * FROM bans WHERE license = @license', {['@license'] = QBCore.Functions.GetIdentifier(source, 'license')})
+
+	if result[1] ~= nil then
+		if os.time() < result[1].expire then
+			retval = true
+
+			local timeTable = os.date("*t", tonumber(result.expire))
+
+			message = "You have been banned from the server:\n" .. result[1].reason .. "\nYour ban expires " .. timeTable.day .. "/" .. timeTable.month .. "/" .. timeTable.year .. " " .. timeTable.hour .. ":" .. timeTable.min .. "\n"
```
Comment (FreakOutModz) [inline server/functions.lua:271]: For better order: `message = "You have been banned from the server:\n" .. result[1].reason .. "\nYour ban expires " .. timeTable.month .. "/" .. timeTable.day .. "/" .. timeTable.year .. " " .. timeTable.hour .. ":" .. timeTable.min .. "\n"`

Using date and time notation in the United States correctly.

File: server/functions.lua [hunk 3]
```diff
@@ -140,34 +164,40 @@ QBCore.Functions.IsWhitelisted = function(source)
...
 
 QBCore.Functions.RemovePermission = function(source)
 	local Player = QBCore.Functions.GetPlayer(source)
-	if Player ~= nil then 
-		QBCore.Config.Server.PermissionList[QBCore.Functions.GetIdentifier(source, 'license')] = nil	
-		exports['ghmattimysql']:execute('DELETE FROM permissions WHERE license=@license', {['@license'] = QBCore.Functions.GetIdentifier(source, 'license')})
+
+	if Player ~= nil then
+		QBCore.Config.Server.PermissionList[QBCore.Functions.GetIdentifier(source, 'license')] = nil
+
```
Comment (FreakOutModz) [inline server/functions.lua:198]: You can remove this blank space, less lines at the end.

File: server/functions.lua [hunk 4]
```diff
@@ -140,34 +164,40 @@ QBCore.Functions.IsWhitelisted = function(source)
 	else
 		rtn = true
 	end
+
```
Comment (FreakOutModz) [inline server/functions.lua:167]: You can remove this blank space.

File: server/commands.lua [hunk 5]
```diff
@@ -50,27 +58,36 @@ QBCore.Commands.Add("tp", "TP To Player or Coords (Admin Only)", {{name="id/x",
 	end
 end, "admin")
 
-QBCore.Commands.Add("addpermission", "Give Player Permissions (God Only)", {{name="id", help="ID of player"}, {name="permission", help="Permission level"}}, true, function(source, args)
+QBCore.Commands.Add("addpermission", "Give Player Permissions (God Only)", {
```
Comment (FreakOutModz) [inline server/commands.lua:61]: Should be `Give player permissions (owner only)`

File: client/events.lua [hunk 6]
```diff
@@ -1,58 +1,62 @@
...
+
 		local success = false
 		local blipFound = false
 		local blipIterator = GetBlipInfoIdIterator()
 		local blip = GetFirstBlipInfoId(8)
 
 		while DoesBlipExist(blip) do
 			if GetBlipInfoIdType(blip) == 4 then
-				cx, cy, cz = table.unpack(Citizen.InvokeNative(0xFA7C7F0AADF25D09, blip, Citizen.ReturnResultAnyway(), Citizen.ResultAsVector())) --GetBlipInfoIdCoord(blip)
+				cx, cy, cz = table.unpack(Citizen.InvokeNative(0xFA7C7F0AADF25D09, blip, Citizen.ReturnResultAnyway(), Citizen.ResultAsVector())) -- GetBlipInfoIdCoord(blip)
```
Comment (FreakOutModz) [inline client/events.lua:55]: Comment can be remove

File: server/player.lua [hunk 7]
```diff
@@ -216,183 +230,231 @@ QBCore.Player.CreatePlayer = function(PlayerData)
...
-		reason = reason ~= nil and reason or "unkown"
+		reason = reason ~= nil and reason or "unknown"
 		local moneytype = moneytype:lower()
 		local amount = tonumber(amount)
-		if amount < 0 then return end
+
+		if amount < 0 then
+			return
+		end
+
```
Comment (FreakOutModz) [inline server/player.lua:272]: Method used was the correct one. Your way simply add more lines.

File: server/player.lua [hunk 8]
```diff
@@ -216,183 +230,231 @@ QBCore.Player.CreatePlayer = function(PlayerData)
...
 	self.Functions.SetMoney = function(moneytype, amount, reason)
-		reason = reason ~= nil and reason or "unkown"
+		reason = reason ~= nil and reason or "unknown"
 		local moneytype = moneytype:lower()
 		local amount = tonumber(amount)
-		if amount < 0 then return end
+
+		if amount < 0 then
+			return
+		end
```
Comment (FreakOutModz) [inline server/player.lua:307]: Again, method that was used was way better then this one. This just add more lines, please make the changes.

File: shared.lua [hunk 9]
```diff
@@ -2708,7 +2712,7 @@ QBShared.Vehicles = {
 		["price"] = 2500000,
 		["category"] = "sportsclassics",
 		["hash"] = `ztype`,
-		["shop"] = "pdm",
+		["shop"] = "pdm"
 	},
 ----------MUST BE ENABLED FOR THE BELOW VEHICLES-------------+set sv_enforceGameBuild 2189-------------------------------
```
Comment (FreakOutModz) [inline shared.lua:2717]: You can remove one of the two comments

File: server/events.lua [hunk 10]
```diff
@@ -1,183 +1,205 @@
...
+		deferrals.done('Duplicate Rockstar License Found')
+	else
+		deferrals.done()
+
+		Citizen.Wait(1000)
+
+		TriggerEvent("connectqueue:playerConnect", name, setKickReason, deferrals)
+	end
+
+	-- Add any additional defferals you may need!
```
Comment (FreakOutModz) [inline server/events.lua:66]: You can remove the comment line

**Review by FreakOutModz — COMMENTED** [review]
I didn't check everything, but you can start making this changes for now. Will make the code look better.

**Pull request closed without merging**